### PR TITLE
fix: sdk menu links

### PR DESCRIPTION
**Pull request by vinckr**
- fix: sidebar sdk
- fix: sdk overview filename

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Comment by aeneasr** [comment]
@vinckr build is failing

**Comment by vinckr** [comment]
fixed now @aeneasr 👍

**Comment by aeneasr** [comment]
Keto is still broken for me - try to reload: https://docs-lttcibbbd-ory.vercel.app/docs/keto/sdk/index

**Comment by aeneasr** [comment]
Kratos is also broken for me

File: vercel.json [hunk 1]
```diff
@@ -854,6 +854,26 @@
...
+      "permanent": false
+    },
+    {
+      "source": "/docs/keto/sdk/index",
+      "destination": "https://www.ory.sh/docs/keto/sdk/overview",
+      "permanent": false
+    },
+    {
+      "source": "/docs/keto/sdk",
+      "destination": "https://www.ory.sh/docs/hydra/sdk/overview",
```
Comment (vinckr) [inline vercel.json:874]:
```suggestion
      "destination": "https://www.ory.sh/docs/keto/sdk/overview",
```

**Comment by vinckr** [comment]
> Keto is still broken for me - try to reload: https://docs-lttcibbbd-ory.vercel.app/docs/keto/sdk/index

@aeneasr 

https://docs-djytkh8g3-ory.vercel.app/docs/hydra/sdk/overview
https://docs-djytkh8g3-ory.vercel.app/docs/keto/sdk/overview
https://docs-djytkh8g3-ory.vercel.app/docs/kratos/sdk/overview
all work,

The redirects work in the test deploy but they are leading to pages no broken in the live docs.
https://docs-djytkh8g3-ory.vercel.app/docs/kratos/sdk/index
https://docs-djytkh8g3-ory.vercel.app/docs/kratos/sdk/
https://docs-djytkh8g3-ory.vercel.app/docs/keto/sdk/index
https://docs-djytkh8g3-ory.vercel.app/docs/keto/sdk/ (keto was going to hydra but fixed in latest commit)

**Comment by vinckr** [comment]
The whole seperation does not make too much sense anymore anyway if you use the same SDK for all projects, 
so this SDK section should be updated next and then a SDK page for self-hosting for each project in selfhosted docs.

**Comment by aeneasr** [comment]
Absolutely, that makes sense! The whole SDK sections needs rework and also explanations how to set up the SDKs. Most of the code can probably be copied over from the quickstart examples. The SDK sections is not yet defined im notion, could you help coming up with a structure proposal in notion how the SDK section should be structured

**Pull request merged**